source/change-streams/change-streams.rst

```diff
@@ -946,3 +996,6 @@ Changelog
 +------------+------------------------------------------------------------+
 | 2022-03-25 | Do not error when parsing change stream event documents.   |
 +------------+------------------------------------------------------------+
+| 2022-04-08 | Support returning point-in-time pre and post-images with   |
+|            | fullDocumentBeforeChange and fullDocument.                 |
```

Comment:
```suggestion
|            | ``fullDocumentBeforeChange`` and ``fullDocument``.                 |
```

Reply:
Fixing this manually in order to preserve the table alignment.